lib/openstudio/extension.rb

```diff
@@ -41,6 +41,24 @@ module Extension
     class Extension
       attr_accessor :root_dir
 
+      # max number of datapoints to run
```

Comment:
If these are constant class variables, then yeah, i guess they would be all caps.